### ISSUE TITLE
Music view: prefer to show most recent output file

### ISSUE THREAD

**Issue by sincere-music** [frescobaldi/frescobaldi#1746]
Currently, when multiple output files (`.pdf`) match the LilyPond source file, Music View will default to showing the one that’s alphabetically first. However, I always use a template output-suffix that inserts the date at the end of the filename (YYYY-MM-DD) and thus Music View will show outdated output files, until I manually switch.

The simplest fix for this situation would be to prefer the alphabetically last matching output file, but there may easily be situations where that’s not preferable (when suffix or filename differ by other things than dates). A smarter approach involving modification date of the file may be necessary?

**Comment by jeanas** [comment]
IMHO this is a special case of #1186. Frescobaldi should simply not show output files from previous compilation runs at all.

**Comment by sincere-music** [comment]
Makes sense. I might say that this is a case where the desire to find a comprehensive, fundamental solution for the issue has caused huge delay to solving any of the individual, much simpler things, but then I'm not in the development team, so maybe I shouldn't say anything :)
